src/lucky/redirectable.cr

```diff
@@ -1,19 +1,44 @@
+# Redirect the browser
+#
+# There are multiple ways to redirect inside of an action. The most common ways are to redirect to a `Lucky::Action` class, or a URL/path `String`. Both use the `redirect` method:
+#
+# ```crystal
+# redirect to: Users::Index
+# redirect to: "https://luckyframework.org/"
+# redirect to: "/users"
+# ```
+#
+# By default, the method will set the status code to `302` A.K.A. "Found". If you want to customize the status code, you can pass it directly:
+#
+# ```crystal
+# # create a user
+# redirect to: Users::Index, status: 201
+#
+# # or use the built in enum value
+# redirect to: Users::Index, status: Status::Created
```

Comment:
I would put all the `Status` enum values here, but I think linking to them once they are documented would be better.

Reply:
I think it would help to use 3xx codes here so it is more realistic http://www.restapitutorial.com/httpstatuscodes.html 

Some example enums would be these: https://github.com/luckyframework/lucky/blob/3d15a40d47b8aa16b806c92c3b698ab54af642a0/src/lucky/action.cr#L35-L40